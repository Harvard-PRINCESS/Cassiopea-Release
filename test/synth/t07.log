Command: Synthesize for spec t07.spec output to t07.out; log to t07.log.raw
Output to: t07.out
Log to: t07.log.raw
Synthesis with 0 insts starting
Found dependencies in: 0.041249s
Found value dependencies in: 0.000036s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Synthesis with 0 insts failed with 1 candidates, 1 counterexamples!
Synthesis with 1 insts starting
Found dependencies in: 0.039593s
Found value dependencies in: 0.000111s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Synthesis with 1 insts failed with 1 candidates, 1 counterexamples!
Synthesis with 2 insts starting
Found dependencies in: 0.041992s
Found value dependencies in: 0.000216s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Synthesis with 2 insts failed with 1 candidates, 1 counterexamples!
Synthesis with 3 insts starting
Found dependencies in: 0.041356s
Found value dependencies in: 0.000457s
Iter 1 Guessing...	Guessing Time: ???s
Iter 1 Checking...	Checking Time: ???s
Iter 2 Guessing...	Guessing Time: ???s
Iter 2 Checking...	Checking Time: ???s
Synthesis with 3 insts succeeded! Prog:
(MFC r0)
(XORI r0 0b11110001)
(MTC r0)
	Execution Time: ???s
