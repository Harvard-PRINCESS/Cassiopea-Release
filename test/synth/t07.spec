def interrupts_are_on  -> bool =  *irqen == 0b1
let f: 1 bit = *fpuen
let e: 1 bit = *bigendian
let l: 1 bit = *longmode
frame:
  reg-modify: r0
pre: interrupts_are_on() == false
post: interrupts_are_on() == true &&
   *fpuen == f &&
   *bigendian == e &&
   *longmode == l
