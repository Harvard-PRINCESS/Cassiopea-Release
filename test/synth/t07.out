(MFC r0)
(XORI r0 0b11110001)
(MTC r0)
